db/seeds.rb

```diff
@@ -65,6 +65,25 @@
   )
 end
 
+# The average number of members for an organization in prod is ~2.
+max_org_members = [4, ((num_users - 3) / 6).floor].min
```

Comment:
The maximum of 4 might be too low. I tried it with a few different numbers (up to 20) with larger multipliers and it works fine, so this is just personal preference.